CONTRIBUTING_JS.md

```diff
@@ -184,7 +182,7 @@ We use [`aegir-docs`](https://github.com/dignifiedquire/aegir) for the actual ge
 
 ### Commits
 
-We have very precise rules over how our git commit messages can be formatted. This leads to more readable messages that are easy to follow when looking through the project history.  But also, we use the git commit messages to generate the change log.
+We have very precise guidelines over how our git commit messages can be formatted. This leads to more readable messages that are easy to follow when looking through the project history.  But also, we use the git commit messages to generate the change log.
```

Comment:
I think this is the correct sentiment - I do not enforce this with new/existing contributors and it's trivial to correct in github with a squash+merge.